data/remnant/remnant missions.txt

```diff
@@ -875,10 +879,63 @@ mission "Remnant: Technology Available"
+			label feast
+			`	He waves dismisively. "I am sure you have more questions, but let us leave it there for now. Our meals await, and, well, I need to show you some stuff in regards to our security procedures. So finish your meal, and let's head over to <destination>."`
+				accept
+	on enter
+		conversation
+			`Once you reach space, the prefect interrupts your normal routine. "I'm sure you noticed that your 'license' isn't exactly a normal ID card. There's a reason for that, namely that it has a whole bunch of extra circuitry in it, and includes a couple sensors on the surface. I don't understand everything in it, but the security specialists assure me that it is incredibly resistance to being hacked or compromised."`
+			choice
+				`	"Okay, sounds good."`
+					goto continue
+				`	"How do you know what regular licenses look like?"`
+			`	The old man smiles. "While we might not have access to modern licenses, we do have all our old entertainment programs from before the exile. Some of them included details such as what licenses looked like. While things might have changed in the intervening years, licenses were historically governed by petty bureaucracy, and those are very resistant to change. So it seemed very likely that things have not changed much. And definitely not so much as to resemble ours." He seems amused, but then his smile fades.`
+			`	"Then again, ancestral space never designed their licenses to withstand much. They are incentivized to not be so secure, are they not? Enough money and the right connections, and licenses for anything can be acquired. These" he holds up a Remnant license "not so much." He shakes his head and becomes serious. "Now, your license has a handful of connection pads on the bottom, which can be connected to your ship's IFF. When you are inbound for landing, simply tap this code on the license, and the cube will verify the pattern as well as your fingerprints, a few other biometrics to ensure you are indeed living and concious, and then transmit an encrypted code. Our IFF systems will verify the transmitted signal and respond accordingly."`
+			choice
+				`	"That sounds thorough."`
+					goto thorough
+				`	"So why haven't I been shot down yet?"`
+					goto shotdown
+				`	"What is an 'IFF' system?"`
+			`	"You aren't familiar with IFF?" He asks, looking slighly incredulous. "IFF stands for Identify Friend or Foe, and is the general name for the system that identifies your ship to those around it. It is what determines which ships should be highlighted red or orange on your system map, and provides limited movement detail to allied ships so that targetting systems can fire past you. I do not know about their current practices, but historically the Navy and most other organizations had complex methods of verifying the identify of other ships. So it is not simply a case of having your IFF state that you are a friendly."`
```

Comment:
```suggestion
			`	"You aren't familiar with IFF?" He asks, looking slighly incredulous. "IFF stands for Identify Friend or Foe and is the general name for the system that identifies your ship to those around it. It determines which ships should be highlighted red or orange on your system map and provides limited movement detail to allied ships so that targeting systems can fire past you. I do not know about their current practices, but historically the Navy and most other organizations had complex methods of verifying the identify of other ships. So it is not simply a case of having your IFF state that you are a friendly."`
```